src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -149,11 +149,41 @@ function replaceButtonHtmlAndTooltipText(button, from, to) {
 function expandOrCollapsePanels(expandCollapseButton, panels) {
     const STRING_EXPAND = 'Expand';
     const STRING_COLLAPSE = 'Collapse';
+    const expandCollapseStudentsButton = 'a[id^="collapse-panels-button-section-"]';
+    const expandCollapseTeamButton = 'a[id^="collapse-panels-button-team-"]';
+    const isButtonInExpandMode = $(expandCollapseButton).html().trim().startsWith(STRING_EXPAND);
+    const isCollapsePanelsButton = !panels;
+    /*
+     * When collapsing expandCollapseButton, all panels buttons under it,
+     * (expandCollapseTeamButton and  expandCollapseStudentButton) should be in collapsed state,
+     * so that next time we expand the main panel, we find the expandCollapseTeamButton and
+     * expandCollapseStudentButton in collapsed state.
+     */
+    if (isCollapsePanelsButton && !isButtonInExpandMode) {
+        if ($(expandCollapseStudentsButton).length
+            && $(expandCollapseStudentsButton).html().trim().startsWith(STRING_EXPAND)) {
+            replaceButtonHtmlAndTooltipText(expandCollapseStudentsButton, STRING_EXPAND, STRING_COLLAPSE);
+            $(expandCollapseStudentsButton).trigger('mouseleave');
+        }
+        if ($(expandCollapseTeamButton).length
+            && $(expandCollapseTeamButton).html().trim().startsWith(STRING_EXPAND)) {
+            replaceButtonHtmlAndTooltipText(expandCollapseTeamButton, STRING_EXPAND, STRING_COLLAPSE);
+            $(expandCollapseTeamButton).trigger('mouseleave');
+        }
+    }
+    // When collapsing expandCollapseTeamButton, expandCollapseStudentButton should be in collapsed state.
+    if ($(expandCollapseStudentsButton).length && (expandCollapseButton === $(expandCollapseTeamButton))) {
+        if ($(expandCollapseTeamButton).html().trim().startsWith(STRING_COLLAPSE)
+            && $(expandCollapseStudentsButton).html().trim().startsWith(STRING_EXPAND)) {
```

Comment:
The condition can be combined with the previous if statement?

Reply:
Yes, I'll combine them.